pages/api/ps/reviews/add.ts

```diff
@@ -0,0 +1,64 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { supabase } from '../../supabase'
+import { getServerSession } from "next-auth/next"
+import { authOptions } from "../../auth/[...nextauth]"
+
+type ResponseData = {
+    message: string,
+    error: boolean
+}
+
+export default async function handler(
+    req: NextApiRequest,
+    res: NextApiResponse<ResponseData>
+) {
+    const session = await getServerSession(req, res, authOptions)
+    if (!session) {
+        res.status(400).json({
+            message: 'Unauthorized, Please login and try again',
+            error: true
+        })
+        return
+    }
+
+    const { type, batch, station, review, created_by } = req.body
+
+    if (!type) {
+        res.status(422).json({ message: 'Invalid Request - Type missing', error: true })
+        return
+    } else if (type === 'PS1') {
```

Comment:
We are anyways returning at end of body, probably can use this as an opportunity to flatten the if..else statement.